server/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/GrammarPreprocessorListenerImpl.java

```diff
@@ -201,6 +201,10 @@ private void collectAndAccumulateCopybookData(
       recursiveReplaceStmtStack.add(new ArrayList<>(copyReplacingClauses));
       copyReplacingClauses.clear();
     }
+
+    // Do preprocessor cleanup, before replacements.
+    content = preprocessor.codeCleanUp(uri, content).unwrap(errors::addAll);
```

Comment:
Do I understand correctly that now you apply code clean up twice - here and in `processCopybook` via `preprocessor.process()`?

Reply:
Sorry, I didn't notice splitting the logic in the `TextPreprocessorImpl`. Maybe, it would be better to rename its methods somehow to highlight that one applies the cleanup, and the other - don't.

Reply:
Does processNormalizedCode sound good for the method with process cleaned-up code?

Reply:
If we assume that `process` and `cleanUp` work in a pair, so `processCleanCode` would sound the least confusing.